frontend/src/pages/Feed/Post/component.tsx

```diff
@@ -1,29 +1,35 @@
-import { type postInterface } from '@pages/Feed/Post/interface'
+import {
+  type trackInterface,
+  type postInterface,
+} from '@pages/Feed/Post/interface'
+import { getTrack } from '@services/SpotifyAPI'
 import { Card } from 'antd'
+import { useEffect, useState } from 'react'
 
 const { Meta } = Card
 
 export function Post(props: postInterface): JSX.Element {
+  const [post, setPost] = useState<trackInterface>()
+
+  useEffect(() => {
+    getTrack(`${props.spotifyId}`)
+      .then((res) => {
+        setPost(res)
+      })
+      .catch((err) => {
+        console.log(err)
+      })
+  }, [])
```

Comment:
Does it make sense to include prop.spotifyId into the dependency array of useEffect ?